spec/rails_helper.rb

```diff
@@ -7,6 +7,7 @@
 
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV['RAILS_ENV'] ||= 'test'
+ENV['ENV'] ||= 'test'
```

Comment:
I don't personally like the idea of modifying ENV - it is supposed to be a representation of the external environment variables.  I know the rspec helper does it - but that is bad - I would not copy that to be honest - we use dotenv for this reason